common/src/java/com/zimbra/common/io/TcpServerInputStream.java

```diff
@@ -64,4 +71,13 @@ public String readLine() throws IOException {
             buffer.append((char)ch);
         }
     }
+
+    public void close ()  throws IOException {
+        try {
+           br.close();
+           super.close();
+        } catch (IOException $e) {
+            // @TODO --> handle / log exception
```

Comment:
Let's not leave a @TODO comment here, since there is nothing to do when handing this exception. I would simply ignore the exception and leave a comment in the code to that effect.